<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidgetGeneral</class>
 <widget class="QWidget" name="SettingsWidgetGeneral">
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="rememberLastDatabasesCheckBox">
     <property name="text">
      <string>Remember last databases</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="modifiedExpandedChangedCheckBox">
     <property name="text">
      <string>Mark as modified on expanded state changes</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
